### ISSUE TITLE
math preview: switch in comments (i.e., something after %)

### ISSUE THREAD

**Issue by Romeoly** [SublimeText/LaTeXTools#1036]
Related #914 #1017 

Compared to adding an additional option to `preview_math_mode`, I think it would be great if we can turn it on or off with a switch. Consider two examples:
----------------------------------------------
`%# previewed, or something short like %* or %%, etc.
\[
    This is an equation to be previewed. 
\]`
and
`\[
  This is an ordinary equation not to be previewed. 
\]`
Or sometimes, 
%*
$an-equation-line$ to be viewed.
----------------------------------------------

To summarize, it is very easy to insert a comment like `%comment` in the `tex` file without any influence on the produced PDF output. Thus, it is possible to use the comment as a switch? If I want an equation to be previewed, I simply write append or prepend a `%%' or something like that in the `tex` file. 

I hope I put my request clearly.
